src/main/kotlin/no/skatteetaten/aurora/boober/feature/S3Feature.kt

```diff
@@ -116,7 +116,7 @@ class S3Feature(
             ).singleOrNull()
                 ?.claims ?: emptyList()
 
-            if (claims.isEmpty() || claims.any { it.ownerId == applicationDeploymentId }) null
+            if (claims.isEmpty() || (claims.size == 1 && claims.first().ownerId == applicationDeploymentId)) null
```

Comment:
Tror vi bør gå gjennom denne valideringen. Og se om man kan snu logikken litt. Det vi skal sjekke her er at man skal enten få ingen claims eller 1 claim som er eid av deg selv?

Reply:
Slik jeg ser det er det 3 forskjellige ting som kan gå feil her. 

1. Herkimer kan returnere mer en 1 response, den blir idag svelgt av singleOrNull (tviler på om denne skjer)
2. Man kan få fler enn 1 claim i en response fra herkimer
3. Claimet man får er eid av noen andre en deg. 

Forslår å skrive om singleOrNull til en flatMap eller noe som henter ut alle claims for alle responses. Så guarde tidlig på at du er tom